src/target/move.c

```diff
@@ -305,7 +280,7 @@ void turn_back(void)
 
 	keep_front_wall_distance(CELL_DIMENSION / 2.);
 	disable_walls_control();
-	speed_turn(side, 0);
+	speed_turn(side, speed);
```

Comment:
This is not only refactoring, before the speed to turn_back was always 0. This change impacts on the behaviour ...

Reply:
Yeah, it will turn back faster with a faster speed level, as expected. Before the speed level was not considered and the turn back was executed at the minimum speed always, which probably makes little sense?

Reply:
Ok then